rskj-core/src/test/java/co/rsk/peg/BridgeSupportIT.java

```diff
@@ -192,17 +199,114 @@ void testInitialChainHeadWithoutBtcCheckpoints() throws Exception {
     void testInitialChainHeadWithBtcCheckpoints() throws Exception {
         BridgeConstants bridgeTestNetConstants = BridgeTestNetConstants.getInstance();
 
+        BridgeSupport bridgeSupport = arrangeBridgeSupport(bridgeTestNetConstants,
+            activationsBeforeForks);
+
+        // Force instantiation of blockstore
```

Comment:
why doing this?

Reply:
Doing exactly what's being said. Check this: https://github.com/rsksmart/rskj/blob/8220d123db03376d90b1357ada2d11f65c93de89/rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java#L2573

Basically, we are testing how checkpoints are read and then store in the `RepositoryBtcBlockStoreWithCache`.

Reply:
my concern is: why is this needed? maybe an explanatory comment to add a bit more context

Reply:
Refactored